### PR TITLE
Add shorthand rad list usage example

### PR DESCRIPTION
**Pull request by bjoginapally**
# Description

Add shorthand rad list usage example

## Issue reference

https://github.com/project-radius/radius/issues/3119

Please reference the issue this PR will close: #3119

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Adds necessary unit tests for change
* [ ] Adds necessary E2E tests for change
* [x] Unit tests passing
* [x] Extended the documentation / Created issue for it

### CODE REVIEW

**Review by emily-potyraj — APPROVED** [review]
love it! 🔥

**Review by kachawla — APPROVED** [review]

**Review by kachawla — COMMENTED** [review]
Please wait for @AaronCrawfis to approve as well before merging.

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**
